src/main/tools/linux-sandbox-pid1.cc

```diff
@@ -442,6 +518,119 @@ static int WaitForChild() {
   }
 }
 
+static void MountSandboxAndGoThere() {
+  if (mount(opt.sandbox_root.c_str(), opt.sandbox_root.c_str(), nullptr, MS_BIND | MS_NOSUID, nullptr) < 0) {
+    DIE("mount");
+  }
+  if (chdir(opt.sandbox_root.c_str()) < 0) {
+    DIE("chdir(%s)", opt.sandbox_root.c_str());
+  }
+}
+
+static void CreateEmptyFile() {
+  // This is used as the base for bind mounting.
+  if(CreateTarget("tmp", true) < 0 ){
+      DIE("CreateTarget tmp")
+  }
+  CreateFile("tmp/empty_file");
+}
+
+static void MountDev() {
+  if (CreateTarget("dev", true) < 0) {
+    DIE("CreateTarget /dev");
+  }
+  const char *devs[] = {"/dev/null", "/dev/random", "/dev/urandom", "/dev/zero", NULL};
```

Comment:
`/dev/shm` might be a good idea (should also be writable) - it is required if python code is to be executed and it happen to use `multiprocessing` module.

Reply:
The sandbox would be less hermetic if /dev/shm is bind mounted, since then actions could communicate with each other via /dev/shm.

Would it make sense to require an explicit --sandbox_add_mount_pair for use cases where /dev/shm is desired to be bind mounted?